lib/features/game/views/widgets.dart

```diff
@@ -78,22 +82,28 @@ class HeaderText extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.fromLTRB(16, 30, 16, 0),
-      child: Text(text, style: TextStyle(fontSize: 25, color: KColors.basedBlackColor)),
+    return FittedBox(
+      fit: BoxFit.fitWidth,
+      child: Padding(
+        padding: const EdgeInsets.fromLTRB(16, 30, 16, 0),
+        child: Text(text, style: TextStyle(fontSize: 35, color: KColors.basedBlackColor)),
```

Comment:
that's crazy large fontsize!!! Why would we ever need such a big font?

Reply:
done